devdoc/umockcallrecorder_requirements.md

```diff
@@ -200,3 +201,24 @@ extern int umockcallrecorder_fail_call(UMOCKCALLRECORDER_HANDLE umock_call_recor
 **SRS_UMOCKCALLRECORDER_01_050: [** If index is invalid, umockcallrecorder_fail_call shall return a non-zero value. **]**
 
 **SRS_UMOCKCALLRECORDER_01_051: [** If umockcall_set_fail_call fails, umockcallrecorder_fail_call shall return a non-zero value. **]**
+
+
+## umockcallrecorder_can_call_fail
+
+```c
+extern int umockcallrecorder_can_call_fail(UMOCKCALLRECORDER_HANDLE umock_call_recorder, size_t index, int* can_call_fail);
+```
+
+
+**SRS_UMOCKCALLRECORDER_31_056: [** If umock_call_recorder or call_can_fail is NULL, umockcallrecorder_can_call_fail shall return a non-zero value. **]**
```

Comment:
>umock_call_recorder or call_can_fail [](start = 39, length = 36)

Consider splitting this in 2 requirements. It's a consider as I see the whole file does it the "old" way.